apps/emqx/src/emqx_authentication.erl

```diff
@@ -902,3 +908,13 @@ to_list(M) when is_map(M) -> [M];
 to_list(L) when is_list(L) -> L.
 
 call(Call) -> gen_server:call(?MODULE, Call, infinity).
+
+inc_authenticate_metric(Metric) ->
+    emqx_metrics:inc('client.authenticate'),
```

Comment:
We need increase this metrics at `emqx_access_control.erl`. Because it represents the number of times the `client.authenticate` was executed, instead of `emqx_authn:authenticate/2` was executed.